.github/workflows/ci.yml

```diff
@@ -116,7 +116,7 @@ jobs:
       matrix:
         python: ['3.6', '3.7', '3.8', '3.9', '3.10']
         include:
-          - python: '3.8'
+          - python: '3.8'  # <- not actually used
```

Comment:
@pquentin it's not just that the `python` value is not used — this inclusion does not generate a macOS job at all. This is because it references non-existing factors (which is solved in other matrixes with an empty string).

Reply:
Oh, good catch. We should fix that.

Reply:
Yeah, I noticed it while preparing https://github.com/python-trio/trio/pull/2443

Reply:
I also noticed that those non-existing factors aren't even referenced in steps.